pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
```

Comment:
Is it the installation result with pnpm@8.6.1 ?

Reply:
fixed, the corepack not working..